nomenclature/processor/requiredTS.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+from typing import List, Optional, Union
+
+import yaml
+from pydantic import BaseModel
+
+
+class RequiredTS(BaseModel):
```

Comment:
I'm not a fan of using non-standard abbreviations - maybe call this `RequiredData` (because this is the `data` attribute in an IamDataFrame)?